src/components/StrongPasswordInput/index.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Label from '../Input/label';
+import StyledIconContainer from '../Input/styled/iconContainer';
+import RelativeElement from '../Structural/relativeElement';
+import StyledInput from './styled/input'; // TODO: change to import from index
+import RenderIf from '../RenderIf';
+import HelpText from '../Input/styled/helpText';
+import ErrorText from '../Input/styled/errorText';
+import StrengthBar from './strengthBar';
+import { StyledContainer } from './styled';
+import { useUniqueIdentifier, useErrorMessageId, useLabelId } from '../../libs/hooks';
+import usePasswordState from './hooks/usePasswordState';
+
+/**
+ * StrongPasswordInput component is used to provide feedback about password strength
+ * @category Form
+ */
+const StrongPasswordInput = React.forwardRef((props, ref) => {
+    const {
+        id,
+        className,
+        style,
+        bottomHelpText,
+        error,
+        passwordState,
+        passwordStateLabel,
+        value,
+        placeholder,
+        onChange,
+        disabled,
+        readOnly,
+        tabIndex,
+        onFocus,
+        onBlur,
+        onClick,
+        onKeyDown,
+        maxLength,
+        minLength,
+        icon,
+        required,
+        name,
+        label,
+        hideLabel,
+        iconPosition,
+    } = props;
+
+    const inputId = useUniqueIdentifier('input');
+    const errorMessageId = useErrorMessageId(error);
+    const labelId = useLabelId(label);
+    const status = usePasswordState(passwordState);
+
```

Comment:
We need to implement focus, blur, and click in order to let the user call these methods programmatically. Look at DateTimePicker to see how it gets done in a functional component